bin/pygrb/pycbc_pygrb_efficiency

```diff
@@ -287,35 +283,34 @@ if do_injections:
     sites = [ifo[0] for ifo in ifos]
 
     # Triggers and injections recovered in some form: discard ones at vetoed times
-    found_trigs = ppu.load_injections(found_file, vetoes)
-    found_injs = ppu.load_injections(found_file, vetoes, sim_table=True)
+    inj_file = ppu.load_triggers(found_missed_file, vetoes)
```

Comment:
This is the hdf5 file object, so
```suggestion
    injs = ppu.load_triggers(found_missed_file, vetoes)
```
is more intuitive as a name and once you have this, there is no need to extract columns, you simply navigate the hdf5 object, when needed (e.g., `injs[found/ra][:]` instead of `found_inj['ra'][:]`).